### PR TITLE
Add blueprint to Human Pose Tracking example

### PR DESCRIPTION
**Pull request by abey79**
### What

Add blueprint to Human Pose Tracking. That example was pretty good already, but still a nice touch.

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5612/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5612/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5612/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5612)
- [Docs preview](https://rerun.io/preview/2a0e41c68fa57f01eeccf451bcba4dfe0de2fa7d/docs) 
- [Examples preview](https://rerun.io/preview/2a0e41c68fa57f01eeccf451bcba4dfe0de2fa7d/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: examples/python/human_pose_tracking/main.py [hunk 1]
```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
+    return rrb.Viewport(
+        rrb.Horizontal(
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video", name="Segmentation/reprojection"),
```
Comment (Wumpf) [inline examples/python/human_pose_tracking/main.py:197]: I find the slash in there a bit strange tbh - looks like an entity path again
What do you think of

```suggestion
                rrb.Spatial2DView(origin="video", name="Segmentation & reprojection"),
```

or just `Segmentation` or `Result`
```

File: examples/python/human_pose_tracking/main.py [hunk 2]
```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
```
Comment (Wumpf) [inline examples/python/human_pose_tracking/main.py:193]: I haven't seen this anywhere else yet done with a getter utility. I don't mind per se, but we shouldn't set a random precedence imho.

File: examples/python/human_pose_tracking/main.py [hunk 3]
```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
...
+    return rrb.Viewport(
```
Comment (Wumpf) [inline examples/python/human_pose_tracking/main.py:194]: `rrb.Viewport` is redundant here, you can just pass a `Horizontal` out

File: examples/python/human_pose_tracking/main.py [hunk 4]
```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
...
+        rrb.Horizontal(
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video", name="Segmentation/reprojection"),
+                rrb.Spatial3DView(origin="person", name="3D pose"),
+            ),
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video/rgb", name="Raw video"),
```
Comment (Wumpf) [inline examples/python/human_pose_tracking/main.py:201]: nice touch with the raw input! Description box feels almost a bit crammed being so slender. Makes me realize you lied a little bit on the screenshot? Out of the box there's still a selection panel, not seeing it beeing hidden here

**Review by Wumpf — CHANGES_REQUESTED** [review]

File: examples/python/human_pose_tracking/main.py (hunk 1 above)
Comment (abey79) [reply examples/python/human_pose_tracking/main.py:197]: good point. let's go for `Result`

File: examples/python/human_pose_tracking/main.py (hunk 2 above)
Comment (abey79) [reply examples/python/human_pose_tracking/main.py:193]: Yeah I did that in the release checklist. I'll change it for consistency, but I do find having layout code right in the middle of administrative CLI stuff is kind of weird :)

File: examples/python/human_pose_tracking/main.py (hunk 3 above)
Comment (abey79) [reply examples/python/human_pose_tracking/main.py:194]: 👍🏻

File: examples/python/human_pose_tracking/main.py (hunk 4 above)
Comment (abey79) [reply examples/python/human_pose_tracking/main.py:201]: tuned it a bit.

File: examples/python/human_pose_tracking/main.py (hunk 2 above)
Comment (Wumpf) [reply examples/python/human_pose_tracking/main.py:193]: we might need to talk this out and do a quick pass at the end then again

**Review by Wumpf — APPROVED** [review]

File: examples/python/human_pose_tracking/main.py (hunk 4 above)
Comment (abey79) [reply examples/python/human_pose_tracking/main.py]: And updated screenshot 👍🏻

**Pull request merged**